data/schemas/keyboard.jsonschema

```diff
@@ -406,6 +406,13 @@
                 }
             }
         },
+        "repeat_key": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {"type": "boolean"}
+            }
+        },
```

Comment:
I appreciate those clarifications and the review! Earlier in this PR, @Ikcelaks made a good suggestion that a following PR could add an option to forget capitals from the repeated mods (use case: so that "Aaron" doesn't come out as "AAron"). I made this object with the intention that such an option would eventually go there. I'll go ahead and add a forget capitals option in this PR.